lite2/client.go

```diff
@@ -740,72 +740,66 @@ func (c *Client) sequence(newHeader *types.SignedHeader, newVals *types.Validato
 	return Verify(c.chainID, c.trustedHeader, c.trustedNextVals, newHeader, newVals, c.trustingPeriod, now, c.trustLevel)
 }
 
-// see VerifyHeader
+// see VerifyHeader  NB: although newVals is entered as input actually updates client database with newNextVals
```

Comment:
NB?

Reply:
Note well / Nota Bene

Reply:
let's stick with the commonly accepted codetags (https://en.wikipedia.org/wiki/Comment_(computer_programming)#Tags, https://docs.microsoft.com/en-us/visualstudio/ide/using-the-task-list?view=vs-2015&redirectedfrom=MSDN#customTokens)

Reply:
sure, thanks for pointing them out.